architecture-decisions/0003-cicognara-letsencrypt-path.md

```diff
@@ -0,0 +1,24 @@
+# 1. Cicognara Let's Encrypt Path
+
+Date: 2020-02-11
+
+## Status
+
+Accepted
+
+## Context
+
+There are instances where we will be unable to use our InCommon Federation SSL certificate framework for certs. In instances where a dot org, etc., In those instances Let's Encrypt is an adequate solution. 
+
+The Lets Encrypt expects the SSL certs to be under `/etc/letsencrypt/` which differs from where we place the rest of our SSL certificated
+
+
+## Decision
+
+make sure certificates are under `/etc/letsencrypt/`
+create a directory under `/var/www/letsencrypt/<servicename>`
```

Comment:
What happens with this directory? Is it empty?

Reply:
yes it is written to temporarily when LE is verifying.

Reply:
👍 maybe just add that as the end of the sentence?